libvcx/src/api/utils.rs

```diff
@@ -130,19 +130,8 @@ pub extern fn vcx_agent_update_info(command_handle: CommandHandle,
     };
 
     spawn(move || {
-        match messages::agent_utils::update_agent_info(&agent_info.id, &agent_info.value) {
-            Ok(()) => {
-                trace!("vcx_agent_update_info_cb(command_handle: {}, rc: {})",
-                       command_handle, error::SUCCESS.message);
-                cb(command_handle, error::SUCCESS.code_num);
-            }
-            Err(e) => {
-                error!("vcx_agent_update_info_cb(command_handle: {}, rc: {})",
-                       command_handle, e);
-                cb(command_handle, e.into());
-            }
-        };
-
+        error!("vcx_agent_update_info is not supported anymore");
+        cb(command_handle, error::NOT_READY.code_num);
```

Comment:
I don't think it matters, we won't ever be calling this, in fact we'll probably delete this whole function as well. I didn't want to go that far because then I'd have bother also start removing it from all the wrappers and it's not the point right now.